ultravox/data/datasets.py

```diff
@@ -76,29 +76,27 @@ class DataCollatorForSeq2SeqWithAudio(transformers.DataCollatorForSeq2Seq):
 
     def __call__(self, features, *args, **kwargs):
         audio_values = [f.pop("audio_values", None) for f in features]
-        if self.include_alt_fields:
-            # these fields are hard-coded in the transformer data collator, so they need special handling before calling the super method
-            alt_features = [
-                {
-                    "input_ids": f.pop("alt_input_ids"),
-                    "attention_mask": f.pop("alt_attention_mask"),
-                    "labels": f.pop("alt_labels"),
-                }
-                for f in features
-            ]
```

Comment:
The reason I had removed this stuff was because it was never needed in inference mode. We shouldn't be removing it in here though, else KD training won't work.